src/server.js

```diff
@@ -24,8 +24,8 @@ class MondayServerSdk {
     return await mondayApiClient.execute(params, token);
   }
 
-  oauthToken(code, clientId, clientSecret) {
```

Comment:
@chris-packett If you could put `redirectUri` inside `options` to make room for more arguments in the future that would be nice :) 

Like this:
```
oauthToken(code, clientId, clientSecret, options) {
    return oauthToken(code, clientId, clientSecret, options);
}

// src/services/oauth-service.js
const oauthToken = (code, clientId, clientSecret, options) => {
  let data = { code, client_id: clientId, client_secret: clientSecret };

  if (options?.redirectUri) {
    data.redirect_uri = options.redirectUri;
  }

  return execute(data, null, { url: MONDAY_OAUTH_TOKEN_URL });
};